querying/github.go

```diff
@@ -167,6 +173,23 @@ func (gh *GithubDataSource) processRepoFindings(projects *ProjectCollection, rep
 			finding.Severity = githubSeverities[vuln.SecurityVulnerability.Severity]
 		}()
 	}
+
+	if repo.VulnerabilityAlerts.PageInfo.HasNextPage {
+		var repoQuery repositoryQuery
+		queryVars := map[string]interface{}{
+			"repoName":    githubv4.String(repo.Name),
+			"orgName":     githubv4.String(gh.orgName),
+			"alertCursor": githubv4.String(repo.VulnerabilityAlerts.PageInfo.EndCursor),
+		}
+		err := gh.GhClient.Query(gh.ctx, &repoQuery, queryVars)
+		if err != nil {
+			return err
+		}
+
+		log.Info().Any("alertCursor", queryVars["alertCursor"]).Msg("Querying for more vulnerabilities for a repository.")
```

Comment:
```suggestion
		log.Info().Str("repoName", repo.Name).Any("alertCursor", queryVars["alertCursor"]).Msg("Querying for more vulnerabilities for a repository.")
```
Let's log the repo name as well, so we know what we're iterating on 😄

Reply:
Done!